src/api/decommission-service/helpers/scale-ecs-to-zero-in-environment.test.js

```diff
@@ -1,13 +1,23 @@
-import { scaleEcsToZero } from '~/src/api/decommission-service/helpers/scale-ecs-to-zero.js'
-import { scaleEcsToZeroInEnvironment } from '~/src/api/decommission-service/helpers/scale-ecs-to-zero-in-environment.js'
-import { getEntity } from '~/src/helpers/portal-backend/get-entity.js'
-import { lookupTenantService } from '~/src/helpers/portal-backend/lookup-tenant-service.js'
+import { describe, beforeAll, expect, test, vi } from 'vitest'
 
-jest.mock('~/src/helpers/portal-backend/get-entity.js')
-jest.mock('~/src/api/decommission-service/helpers/scale-ecs-to-zero.js')
-jest.mock('~/src/helpers/portal-backend/lookup-tenant-service.js')
+vi.mock('../../../helpers/oktokit/oktokit.js', () => ({
+  octokit: vi.fn(),
+  graphql: vi.fn()
+}))
 
-const logger = { info: jest.fn(), warn: jest.fn() }
+const getEntity = vi.fn()
+vi.mock('../../../helpers/portal-backend/get-entity.js', () => ({ getEntity }))
+
+const scaleEcsToZero = vi.fn()
+vi.mock('./scale-ecs-to-zero.js', () => ({
```

Comment:
If you are mocking the entire file it can just be pretty sure these can just be `vi.mock('../../../helpers/portal-backend/get-entity.js')`